src/segments/__main__.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals, print_function, division  # pragma: no cover
+
+from segments.cli import main  # pragma: no cover
+
+
+if __name__ == '__main__':  # pragma: no cover
```

Comment:
Nice one (especially with multiple interpreters it is quite a benefit to be able to do `python -m <packagename>` to execute the cli! How about renaming `cli.py` to `__main__.py` (AFAIU we one can regard this as the expected name for the cli entry point as well as the implementation in case it fits)

Reply:
But what would I call the test module? `test____main__.py`? :)

Reply:
aah right, but test_main.py would be okay in this case, i guess?